soc/arm/xenvm/mmu_regions.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <devicetree.h>
+#include <sys/util.h>
+#include <arch/arm/aarch64/arm_mmu.h>
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 0),
+			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 0),
+			      MT_DEVICE_nGnRnE | MT_RW | MT_NS),
```

Comment:
Probably this should be made more explicit but using `MT_xx` is frowned upon. Please switch to use one of the `MT_P_xx_U_xx` macros (`MT_P_RW_U_RW` probably in your case).